### PR TITLE
Some parts of ColonyDealer aren't used.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/colonies/ColonyDealer.ts [hunk 1]
```diff
@@ -81,22 +81,17 @@ export function loadColoniesFromJSON(colonies: Array<SerializedColony>): Array<C
 }
 
 export class ColonyDealer {
-  public coloniesDeck: Array<Colony> = [];
```
Comment (kberg) [inline src/colonies/ColonyDealer.ts:84]: coloniesDeck isn't used anywhere.

File: src/colonies/ColonyDealer.ts [hunk 2]
```diff
@@ -81,22 +81,17 @@ export function loadColoniesFromJSON(colonies: Array<SerializedColony>): Array<C
...
-  public shuffle(cards: Array<Colony>): Array<Colony> {
+  private shuffle(cards: Array<Colony>): Array<Colony> {
     const deck: Array<Colony> = [];
     const copy = cards.slice();
     while (copy.length) {
       deck.push(copy.splice(Math.floor(Math.random() * copy.length), 1)[0]);
     }
     return deck;
   }
-  public discard(card: Colony): void {
```
Comment (kberg) [inline src/colonies/ColonyDealer.ts:95]: discard isn't used anywhere. discardedColonies is directly accessed.

File: src/colonies/ColonyDealer.ts [hunk 3]
```diff
@@ -105,11 +100,9 @@ export class ColonyDealer {
...
       colonyTiles.forEach((e) => allowList.push(e.colonyName));
     }
-    if (players === 1) {
-      count = 4;
-    } else if (players === 2) {
-      count = 5;
-    }
+    // Two-player games and solo games get one more colony.
+
+    const count: number = (players + 2) + (players <= 2 ? 1 : 0);
```
Comment (kberg) [inline src/colonies/ColonyDealer.ts:105]: Simpler than the code above.

File: src/colonies/ColonyDealer.ts [hunk 4]
```diff
@@ -81,22 +81,17 @@ export function loadColoniesFromJSON(colonies: Array<SerializedColony>): Array<C
 }
 
 export class ColonyDealer {
-  public coloniesDeck: Array<Colony> = [];
   public discardedColonies: Array<Colony> = [];
 
-  public shuffle(cards: Array<Colony>): Array<Colony> {
+  private shuffle(cards: Array<Colony>): Array<Colony> {
```
Comment (kberg) [inline src/colonies/ColonyDealer.ts:86]: shuffle is private

**Review by nwai90 — APPROVED** [review]

**Pull request merged**